### PR TITLE
vendor: refresh for OperatorHub config API changes

### PR DESCRIPTION
**Pull request by aravindhp**
operatorhubs.config.openshift.io was updated in https://github.com/openshift/api/pull/405. This PR updates client-go to account for those changes by running:
- `make update-deps`
- `make generate`

### CODE REVIEW

File: vendor/github.com/openshift/api/config/v1/types_scheduling.go [hunk 1]
```diff
@@ -6,7 +6,8 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// Scheduler holds cluster-wide information about Scheduler.  The canonical name is `cluster`
+// Scheduler holds cluster-wide config information to run the Kubernetes Scheduler
+// and influence its placement decisions. The canonical name for this config is `cluster`.
```
Comment (ravisantoshgudimetla) [inline vendor/github.com/openshift/api/config/v1/types_scheduling.go]: scheduler changes LGTM

/lgtm

**Comment by derekwaynecarr** [comment]
mechanical bump
/approve

**Comment by derekwaynecarr** [comment]
actually, lets hold this bump until the final api shape is merged in https://github.com/openshift/api/pull/409

**Comment by aravindhp** [comment]
@derekwaynecarr https://github.com/openshift/api/pull/409 has merged and this PR has been updated to account for that.

**Pull request merged**